Rocket.Chat/Controllers/Chat/ChatSections/MessageSection.swift

```diff
@@ -94,12 +110,12 @@ final class MessageSection: ChatSection {
             ).wrapped)
         }
 
-        if !object.isSequential {
+        if !object.isSequential && !object.message.text.isEmpty {
```

Comment:
Should you use `shouldAppendMessageHeader` here?

Reply:
Sure